Gems/AtomLyIntegration/EditorModeFeedback/Assets/Shaders/EditorModeCommon.azsli

```diff
@@ -44,6 +44,75 @@ partial ShaderResourceGroup PassSrg : SRG_PerPass_WithFallback
     //! The final blend amount that is used to scale the calculated blend values.
     float m_finalBlendAmount;
     
+    ///////////////////////////////////
+    //! EditorModeDesaturation related data
+    //! Amount of desaturation to apply.
+    float m_desaturationAmount;
+    ///////////////////////////////////
+    
+    ///////////////////////////////////
+    //! EditorModeOutline related data
+    //! Outline line thickness.
+    float m_lineThickness;
+
+    //! Outline line color.
+    float4 m_lineColor;
+
+    //! Outline style to use (0 = OutlineStyle::Always, 1 = OutlineStyle::Visible)
+    uint m_outlineStyle;
+    ///////////////////////////////////
+      
+    ///////////////////////////////////
+    //! EditorModeBlur related data
+    //! Width of kernel to apply box blur effect.
+    float m_kernelHalfWidth;
+
+    //! The direction to perform the blur (0 = horizontal, 1 = vertical).
+    uint m_direction;
+    ///////////////////////////////////
+     
+    ///////////////////////////////////
+    //! EditorModeTint related data
+    //! Amount of tint to apply.
+    float m_tintAmount;
+
+    //! Color of tint to apply.
+    float4 m_tintColor;
+    ///////////////////////////////////
+        
+    ///////////////////////////////////
+    //! EditorModeDepthTransition related data
+    /* -----------------
+        \      E      /
+         \#####|#####/
+          \####|####/
+           \###|###/
+            \##|##/
+             \-S-/
+              \ /
+               V
+
+        V = View position
+        S = Depth transition start
+        E = Depth transition end
+
+        When E is a non-zero value, the binary mask value will be transformed into a value from [m_minDepthTransitionValue, 1.0]
+        according to the fragment's depth position in the transition band (#) spanning from S to E. This is to allow feedback
+        effects to be scaled according to the distance of non-mask fragments to the viewer.
+    */
+
```

Comment:
nit: something to make it visually obvious this is a multiline comment. I'd sugggest either a `*` per line or use `//` on every line.

Reply:
Given that we need this to go into stab today I wont make this change in this PR but can make it in dev branch later.